### DISCUSSION TITLE
Cannot load js from vercel from my site by using script tag

### DISCUSSION THREAD

**Discussion by lukexiongjian** [vercel/community#6308] [Help] [answered]
### Summary

I tried to load a JS file from Vercel by using a script tag in an HTML file, but it did not work.

Status Code:
401 Unauthorized
Referrer Policy:
strict-origin-when-cross-origin

I don't know what could I do to make it possible to load JS file.

### Steps to Reproduce

I tried to load a JS file from Vercel by using a script tag in an HTML file, but it did not work.

**Comment by amyegan** [comment, marked as answer]
Hey @lukexiongjian. You can learn more about cross-origin resource sharing and how to enable CORS here: https://vercel.com/guides/how-to-enable-cors
